doc(serverless): add workaround for return data that is array/list (#5570)
influxdata/docs-v2: content/influxdb/cloud-serverless/query-data/troubleshoot-and-optimize/troubleshoot.md

- Your data falls outside the time range (or other conditions) in the query--for example, the InfluxQL `SHOW TAG VALUES` command uses a default time range of 1 day.
- The query (InfluxDB server) timed out.
- The query client timed out.

> - The query return type is not supported by the client library. For example, array / list type may not be supported, in this case convert the result into a string using `array_to_string()`. For example,

Reviewer: ```suggestion
- The query return type is not supported by the client library.
  For example, array or list types may not be supported.
  In this case, use `array_to_string()` to convert the array value to a string--for example:
```

{{% code-tab-content %}}

```sql
SELECT array_agg([1, 2, 3])
```

can be rewrote into

```sql
SELECT array_to_string(array_agg([1, 2, 3]), ', ')
```

> {{% /code-tab-content %}}

Reviewer: ~~~suggestion
```sql
SELECT array_to_string(array_agg([1, 2, 3]), ', ')
```
~~~
